Add draw_equator and draw_prime_meridian to visualization (#6251)
sunpy/sunpy: changelog/6251.feature.rst

> Add draw module which includes draw_equator and draw_prime_meridian functions to `sunpy.visualization`.

Reviewer: ```suggestion
Created the `sunpy.visualization.draw` module which includes `.draw_equator` and `.draw_prime_meridian` functions.
```
I forget if this will link correctly.

Reviewer 2: ```suggestion
Created the `sunpy.visualization.draw` module which includes :func:`~sunpy.visualization.draw_equator` and :func:`~sunpy.visualization.draw_prime_meridian` functions.
```

Created the `sunpy.visualization.draw` module which includes 

> :func:`~sunpy.visualization.equator` and 
> :func:`~sunpy.visualization.prime_meridian` functions.

Reviewer 2: ```suggestion
:func:`~sunpy.visualization.draw.equator` and 
:func:`~sunpy.visualization.draw.prime_meridian` functions.
```
